dotnet/Firely.Server.MessageSender/appsettings.json

```diff
@@ -4,5 +4,18 @@
       "Default": "Information",
       "Microsoft.Hosting.Lifetime": "Information"
     }
+  },
+  "ImportOptions": {
+    "FhirVersion": "R4",
+    "ImportDirectory": "../Firely.Server.MessageSender.Test/data/"
+  },
+  "PubSub": {
+    "MessageBroker": {
+        "Host": "http://localhost:4317", // The URL where the message broker can be found
+        // "Username": "guest", // RabbitMQ username
```

Comment:
Consider uncommenting the username and password as we expect to use credentials to access rabbitmq